core/modules/field_group/field_group.field_ui.inc

```diff
@@ -0,0 +1,830 @@
+      $table[$name]['format']['#cell_attributes'] = array('colspan' => $display_overview ? 3 : 2);
+      $table[$name]['format']['format_settings'] = array(
+        '#type' => 'container',
+        '#attributes' => array('class' => array('field-formatter-settings-edit-form')),
+        '#parents' => array('fields', $name, 'format_settings'),
+        '#weight' => -5,
+        'label' => array(
+          '#markup' => t('Field group format:') . ' ' . $type,
+        ),
+        'actions' => array(
+          '#type' => 'actions',
+          'save_settings' => $base_button + array(
+            '#type' => 'submit',
+            '#name' => $name . '_formatter_settings_update',
+            '#value' => t('Update'),
+            '#op' => 'update',
+          ),
+          'cancel_settings' => $base_button + array(
+            '#type' => 'submit',
+            '#name' => $name . '_formatter_settings_cancel',
+            '#value' => t('Cancel'),
+            '#op' => 'cancel',
+            // Do not check errors for the 'Cancel' button.
+            '#limit_validation_errors' => array(),
+          ),
+        ),
+      );
+      $table[$name]['#attributes']['class'][] = 'field-formatter-settings-editing';
+      $table[$name]['format']['type']['#attributes']['class'] = array('element-invisible');
+    }
+    else {
+      // After saving, the settings are updated here as well. First we create
+      // the element for the table cell.
+      $table[$name]['settings_summary'] = array('#markup' => '');
+      if (!empty($group->format_settings)) {
+        $summary = field_group_format_settings_summary($name, $group);
+        $table[$name]['settings_summary'] = $summary;
+      }
+
+      $destination = _field_ui_bundle_admin_path($form['#entity_type'], $form['#bundle']);
+      $operations = array(
+        'configure' => array(
+          'title' => t('Configure'),
+          'href' => $params->admin_path . '/groups/' . $name . '/configure/form',
+          'query' => array('destination' => $destination . '/fields'),
+        ),
+        'delete' => array(
+          'title' => t('Delete'),
+          'href' => $params->admin_path . '/groups/' . $name . '/delete/form',
+          'query' => array('destination' => $destination . '/fields'),
+        ),
+      );
+
+      if ($display_overview) {
+        $configure_href = $params->admin_path . '/groups/' . $name . '/configure/' . $params->mode;
+        $configure_destination = $destination . '/display/' . $params->mode;
+        $delete_href = $params->admin_path . '/groups/' . $name . '/delete/' . $params->mode;
+        $delete_destination = $destination . '/display/' . $params->mode;
+
+        $operations['configure']['href'] = $configure_href;
+        $operations['configure']['query']['destination'] = $configure_destination;
+        $operations['delete']['href'] = $delete_href;
+        $operations['configure']['query']['destination'] = $delete_destination;
```

Comment:
Since `$destination` is the same for both operations, we could simplify this to:
```suggestion
        $configure_href = $params->admin_path . '/groups/' . $name . '/configure/' . $params->mode;
        $delete_href = $params->admin_path . '/groups/' . $name . '/delete/' . $params->mode;
        $destination .= '/display/' . $params->mode;

        $operations['configure']['href'] = $configure_href;
        $operations['configure']['query']['destination'] = $destination
        $operations['delete']['href'] = $delete_href;
        $operations['delete']['query']['destination'] = $destination
```
Either way, the last `$operations['configure']['query']['destination']` seems to be a copy/paste error and should be `$operations['delete']['query']['destination']` instead.